src/System.CommandLine.Tests/ParsingValidationTests.cs

```diff
@@ -495,7 +495,11 @@ public async Task A_custom_validator_added_to_a_global_option_is_checked(string
         {
             var handlerWasCalled = false;
 
-            var globalOption = new Option<int>("--value");
+            var globalOption = new Option<int>("--value")
```

Comment:
Do we want to rename this local since we are removing the global concept?

Reply:
I was thinking about it, but I was unable to come up with a better name. `appliesToSelfAndChildrenOption`? `inheritableOption`?